build/webpack/webpack.extension.web.config.js

```diff
@@ -103,7 +103,11 @@ const config = {
                 platform: JSON.stringify('web')
             }
         }),
-        new CleanTerminalPlugin()
+        new CleanTerminalPlugin(),
+        new webpack.IgnorePlugin({
```

Comment:
Should we do this for the desktop extension too?

Reply:
Good point.